test_runner/regress/test_tenants_with_remote_storage.py

```diff
@@ -345,80 +345,6 @@ def test_tenant_upgrades_index_json_from_v0(
 # FIXME: test index_part.json getting downgraded from imaginary new version
 
 
-@pytest.mark.parametrize("remote_storage_kind", [RemoteStorageKind.LOCAL_FS])
-def test_tenant_ignores_backup_file(
```

Comment:
Hmm, so this test tests that we can deal with spurious `*.0.old` files appearing in the list of layers. I presume you checked that we don't have any of those in any `index_part.json` files in production anymore?

Reply:
Yes, there are no such files anymore in the remaining json files.
Ping me for an archive with all such files (it's relatively small) or for the command to create them, if interested.